go/libkb/util_windows.go

```diff
@@ -184,3 +188,47 @@ func RemoteSettingsRepairman(g *GlobalContext) error {
 	}
 	return nil
 }
+
+// Notify the shell that the thing located at path has changed
+func notifyShell(path string) {
+	shChangeNotifyProc.Call(
+		uintptr(0x00002000), // SHCNE_UPDATEITEM
+		uintptr(0x0005),     // SHCNF_PATHW
+		uintptr(unsafe.Pointer(syscall.StringToUTF16Ptr(path))),
+		0)
```

Comment:
NewLazySystemDLL is the best we can do for now I believe. Many Windows APIs are wrapped by various packages but not all, and they all do this kind of thing under the covers anyway.

Note this code just got moved, it's not new, it's in production already.